query-engine/connectors/sql-query-connector/src/ordering.rs

```diff
@@ -38,7 +38,11 @@ impl OrderByBuilder {
                     self.build_order_aggr_scalar(order_by, needs_reversed_order, ctx)
                 }
                 OrderBy::ToManyAggregation(order_by) => self.build_order_aggr_rel(order_by, needs_reversed_order, ctx),
+                #[cfg(any(feature = "postgresql", feature = "mysql"))]
                 OrderBy::Relevance(order_by) => self.build_order_relevance(order_by, needs_reversed_order, ctx),
+                /// Unreachable here since it's way more annoying to feature-gate the enum variant
+                #[cfg(not(any(feature = "postgresql", feature = "mysql")))]
+                OrderBy::Relevance(_) => unreachable!(),
```

Comment:
Should `OrderBy::Relevance` be feature gated?

Reply:
Hmm yes that's tough, I'd prefer to feature gate the enum but I can see that cascading into even more changes, so I'll let you judge what feels best.

Reply:
I think having this `unreachable!` is worth it, gating the variant requires changes across multiple files and doesn't require enabling any database connectors.